UefiPayloadPkg/SPI/own.c

```diff
@@ -0,0 +1,8 @@
+#include <Include/PiDxe.h>
+#include "own.h"
+
+VOID *memset (VOID *ptr, int value, __SIZE_TYPE__ num ) {
```

Comment:
There are lots of memset implementation in EDK2, choose one from MdePkg/Library/BaseMemoryLib*